src/raftstore/store/store.rs

```diff
@@ -287,6 +293,18 @@ impl<T, C> Store<T, C> {
         Ok(())
     }
 
+    fn clean_up_meta(&mut self, wb: &mut WriteBatch, region: &metapb::Region) {
```

Comment:
We have a function named `clean_up` already. Maybe rename two method to `clear_meta` and `clear_data` respectively?